examples/provider/dumps/example.json

```diff
@@ -94,6 +94,9 @@
     "trust_marks_issuers": {},
     "entity_type": "openid_provider",
     "metadata": {
+      "federation_entity": {
+        "federation_resolve_endpoint": "http://127.0.0.1:8002/oidc/op/resolve"
```

Comment:
can I ask you to put also in federation_entity the following members?

organization_name
homepage_uri
policy_uri
logo_uri
contacts